src/pages/sectionMeetingCsv/sectionMeetingCsv.js

```diff
@@ -0,0 +1,32 @@
+const router = require('express').Router({ mergeParams: true })
+const { sqlLoader } = require('@prairielearn/prairielib')
+const { Parser } = require('json2csv')
+const dbDriver = require('../../dbDriver')
+const asyncErrorHandler = require('../../asyncErrorHandler')
+const checks = require('../../auth/checks')
+
+const sql = sqlLoader.loadSqlEquiv(__filename)
+const parser = new Parser()
+
+router.get(
+  '/',
+  asyncErrorHandler(async (req, res, _next) => {
+    if (!(await checks.isLoggedIn(req))) {
+      res.redirect('/login') // TODO: redirect back after login
+      return
+    }
+
+    const result = await dbDriver.asyncQuery(
+      sql.select_swipes_join_section_meetings,
+      { sectionMeetingId: req.params.sectionMeetingId }
+    )
+    if (result.rows.length === 0) {
```

Comment:
If you truly want to special-case a section meeting with no swipes (do we? why not just return a CSV with just headers?), at least call `next()` so the request doesn't hang indefinitely.

Reply:
agree that header-only is probably better.

Reply:
I generate the headers from the JSON object and was too lazy to bother writing in the default headers. Can do that now.